src/main/java/org/folio/consortia/domain/converter/TenantConverter.java

```diff
@@ -0,0 +1,17 @@
+package org.folio.consortia.domain.converter;
+
+import org.folio.consortia.domain.entity.Tenant;
+
+public class TenantConverter {
```

Comment:
We can leaverage build in Spring capabilities to create custom converters by implementing Converter interface

Reply:
Ok, I will do